api/envoy/config/route/v3/route_components.proto

```diff
@@ -1085,6 +1085,25 @@ message RetryPolicy {
     google.protobuf.Duration max_interval = 2 [(validate.rules).duration = {gt {}}];
   }
 
+  message RateLimitedRetryBackOff {
+    // Specifies the rate limited retry reset headers (like ``Retry-After`` or
+    // ``X-RateLimit-Reset``) to match against the response. If a request is
+    // being retried and the response contains one of these headers the value
+    // of the header will be used as the retry back off interval, with jitter added.
+    //
+    // The value of the header must be either:
+    //  1. An interval in seconds as an integer, for example: ``120``.
+    //  2. A unix timestamp as an integer, for example: ``1595320702``. The timestamp
+    //     must be in the future relative to the current system time.
+    repeated HeaderMatcher reset_headers = 1;
```

Comment:
I'm wondering if there is a way to explain this with less jargon to the user. We're basically just taking whatever he backend tells us and using this as the duration for next retry?

What happens when there are multiple retries and the backend only specifies a retry for the first?